webapp/src/App.tsx

```diff
@@ -48,10 +48,12 @@ function App({ hubPort, autoReconnect }: AppProps) {
             <div className="wrap">
                 <div className="left-frame" id="gap">
                     <div className="sidebar-buttons">
-                        <MenuLeft
-                            selectedMode={hubState.daphbot_mode}
-                            onModeChange={handleModeChange}
-                        />
+                        {hubState.hubConnStatus === "online" && (
```

Comment:
I want to undo this.  Whenever there is a hiccup with webapp<-->central the dialog will close and reset.  This happened to me a couple of random times while testing.  

 I think instead pass something like `isConnected` to MenuLeft and have it decide what to do when not.  The VidViewerDialog is not dependent on central_hub data.   It uses the vision service's REST API to retrieve the recorded video files.